int32 markerId
---
bool found
---
float32 time_elapsed
